README.md

```diff
@@ -179,13 +178,51 @@ pip3 install torch_xla[tpuvm]
 
 This is only required on Cloud TPU VMs.
 
-## Performance Profiling and Auto-Metrics Analysis
```

Comment:
why is Performance Profiling section remode?

Reply:
I can move the link to the Cloud documentation with the rest of the tutorials. The colab notebook is deprecated, and I'm not sure the other two links are actually instructive.